heron/packing/src/java/org/apache/heron/packing/roundrobin/RoundRobinPacking.java

```diff
@@ -288,7 +330,7 @@ private ByteAmount getContainerRamPadding(List<TopologyAPI.Config.KeyValue> topo
     }
 
     for (int i = 1; i <= numContainer; ++i) {
-      allocation.put(i, new ArrayList<InstanceId>());
+      allocation.put(i, new ArrayList<>());
```

Comment:
this change seems to be unnecessary?

Reply:
The IDE suggests the type declaration here is unnecessary. I figure just to clean it up

Reply:
kk. sgtm.